src/containers/chat/sidebar-grouped-content/SidebarGroupedContent.styles.ts

```diff
@@ -0,0 +1,6 @@
+export const styles = {
+  groupedContent: {
+    container: { p: '24px 0 0 0' },
```

Comment:
Do you need this p: '24px 0 0 0'?
Use pt: 24px.

Reply:
by default this container has p: '34px', so I need to rewrite it